system-parachains/bridge-hubs/bridge-hub-polkadot/src/bridge_to_kusama_unstuck.rs

```diff
@@ -0,0 +1,97 @@
d90c4069ab003aa4f15699e253fae470100000000000000840e02cf09693432d7a2e2f426009154b83d64824fe9d5960959635e8afb1e370100000000000000ff95b0b84ad21101f39414cc188fa680b950e6b0275a547265d1913e0a882edc01000000000000006ef7d7b23620a1b8d056dccbc113bc1141d089e12a0bda2acab6162d23cea97c0100000000000000b0e79361a461c2b55d5895d3ea11d803df48909ca3c6b1f082d8a4488bc8f068010000000000000043ee65b2ac3f8cd6eaabfe784e75c9c8763bb262da6e5c89331440ab6adcba7b01000000000000006f7153b550fad407bb24c617c790390ac5e00cdb6dde02a9947983ddb2b5f395010000000000000051dc7faf613dc11c9d2ed86db230a1b119b89dacb7e50b8902be6b90f2c6d92901000000000000004155b38bb22d421d48b07f1bad1d82b68dfca2887c56aa4cb513350afdca4b4b01000000000000001ebc598b563a76fcf05a3c4fd25bc1c8496dfd44d73d01934d5a2fdc0d3f72ac0100000000000000ef7c230fbed7284e016b3c9eccad304d1e214c926763c1fc7979734b273f335401000000000000008dea3098e375a1ae6d6bf328b6291c3f3c25b6874efdbe796d78aa8971acb6a601000000000000004d30d35cc1a0c2a91a73d2b168047fd9e2339c45d4f0a96576fb258511aac3cc010000000000000073269b6e07b97fda590681638536929e2110144cf3618ee7c46fe85de37fc52c01000000000000008b930c83cff7a218141050d10f125cf6076f1ff9189a3788df840b0643a06b780100000000000000f2553eebe0dae580ff47f2ddd6739409e3a3bdd05015bf4dca15233dec1b42cf01000000000000005c3c02f412be5563e77739b8f2d824222417c18ebf4aeadfc8dec9dbbf7e2cae0100000000000000f27c568848982feeb905d72ce84db24129f074e871b6af9d9a9abe594187b4f00100000000000000caa143d9cb19ea111a3eb681f29910a05d0d517a674e8cad891b47c439acb38f01000000000000009dcbf390a9c1f74845ce4e8b4622003730ddef85da758293342a80c0892c9f14010000000000000095b085fcc40b6245daabb2fe603023b5c0d4b57a2e3273202d4f66b557f510570100000000000000b37a4373cda750c4cd9b4414f816d29638a5c2f18c00f46f4a865070131a73a0010000000000000047bf8f076eccac66229037703e5a910d2f96aa50d82b2cbe6e1763986a08111f0100000000000000c8c53762490c91aea43491f500a72843e4cc37bb0981867b50c0eb396a79242e01000000000000000899b4434b3ce0818b21838a7765ed2a71b3a5bacb7649be04feeedc4f005ad00100000000000000622a3cfc2a1a3ba6519453fb21c7de1a3435488c89d910f46dc0aa281f38ef6801000000000000008777bebefe7fcb2db8baa196332a3201ec176c68fc1301f95ef12341c320f4d90100000000000000cf152611e73f53af6abc0a235c6732391b80d20d7abbd8d2ab8de408a96b84ac0100000000000000b625674063911d011dd0ccf274bdf63c744caa9000969de4022bec4bc89db5590100000000000000e3a7997474706f3d9cce4f03f7f0373fbcb527509c18996116b5a5849622f2c10100000000000000717d015fa356c546313c535cf5d3fb5e2515836765b269d5714a51258dc1f6d501000000000000008ade77a41e0ef26e54788c6075a73b0687db325ec479198b69094c03165919a00100000000000000293f0c2b3ec0e4297e71b8101cc630dfa5161484ff20653d344d93e984cfcf1a0100000000000000445a772280570650dbe73611369ab9051c4e7dfdc458ee039c7b1e4908568b44010000000000000086957cee8be58488331b4936a0049e77464aa8ff2255d9d78ec33239cbd77e460100000000000000ac5f1503a9d06b06fbb8062584e045abbe365d65af5ff3382e2ce565760ac2ef010000000000000000000000054241424501010c3374b8e5d05fa21b01d0da59264b8204e070b1d0c36a7c5bc9af657c0b694a8aae15ee2f5a1840cfd8b007a6c9b5b0a46022e3ce4e56f57541173c1c8d7085");
```

Comment:
Can you add one more test that does `Blake256::hash(KUSAMA_HEADER_22780167)` and compares it against the expected hash? This makes it easier to verify this :D

Reply:
Done: https://github.com/polkadot-fellows/runtimes/pull/277/commits/63519e98d00c8f662fffb09a648849b374d8d8a2